controls/SV-205624.rb

```diff
@@ -1,36 +1,33 @@
 control 'SV-205624' do
-  title "Windows Server 2019 must automatically remove or disable temporary user accounts after #{input('temporary_account_period_phrase')}."
```

Comment:
Does this change from `#{input()}` to the value because of using the profile.json file to format the controls?